components/at_mention/at_mention.test.tsx

```diff
@@ -227,10 +229,10 @@ describe('components/AtMention', () => {
                 getBoundingClientRect: () => ({
                     top: 400,
                 }),
-            },
+            } as any,
```

Comment:
there is a type for this defined in the react package: `RefObject<T>` with the generic `T` being the kind of `HTMLElement` (`HTMLDivElement` for example)

I would want to keep the usage of `any` as low as possible.